TealiumAdjustExample/TealiumAdjustExample/adjust.json

```diff
@@ -4,42 +4,36 @@
     "sandbox": true,
     "settings": {
       "log_level": "verbose",
-      "delay_start": 1.0,
       "allow_iad": true,
       "allow_ad_services": true,
       "allow_idfa": true,
-      "app_secret" : "secret123",
-      "app_secret_info_1" : "123",
-      "app_secret_info_2" : "456",
-      "app_secret_info_3" : "789",
-      "app_secret_info_4" : "012",
       "event_buffering_enabled": false,
       "send_in_background": true,
       "url_strategy": "<your url strategy>"
```

Comment:
Minor suggestion: Should we use the new config labels for this?  Using the newly supported `url_strategy_domains`, 'url_strategy_use_subdomain',  `url_strategy_is_residency`?

Reply:
I would say no, as we would suggest people to still use the old one. There's really no reason to use the other one other than potentially make a mistake.